wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsplugins/model/PluginFQN.java

```diff
@@ -53,4 +57,28 @@ public String getVersion() {
   public void setVersion(String version) {
     this.version = version;
   }
+
+  @Override
+  public int hashCode() {
```

Comment:
`hashCode` might look simpler like
```java
  @Override
  public int hashCode() {
    return Objects.hash(getRegistry(), getId(), getVersion());
  }
```
But I'm OK with current way as well.

Reply:
The pitfalls of auto-generated code. Fixed.